.gitignore

```diff
@@ -48,6 +49,13 @@ ios/main.jsbundle
 # RVM artifacts
 .ruby-*
 
+# fastlane
```

Comment:
generated by the rn upgrade command.  i choosed to put them to ignore this file in  future upgrades